src/WebApi/wwwroot/api/specification.json

```diff
@@ -109,15 +121,15 @@
         "type": "object",
         "additionalProperties": false,
         "properties": {
-          "id": {
+          "Id": {
```

Comment:
Why are these Pascal case now?  Do we need to add configuration to get it back to camel case?

Reply:
As discussed and explored, the configuring the `"defaultPropertyNameHandling": "CamelCase"` setting in nswag.json is deprecated. 

Controlling the serializer casing now for the openapi/swaggerdoc seems to be quite difficult.

However, the PascalCase 'default' does not seem to be a problem for generating client code (C# / TypeScript).

We will proceed with this new default for now.